launcher/ui/MainWindow.cpp

```diff
@@ -1861,6 +1901,36 @@ void MainWindow::on_actionChangeInstGroup_triggered()
     }
 }
 
+void MainWindow::on_actionCreateShortcut_triggered()
+{
+    if (!m_selectedInstance)
+        return;
+  
+    auto desktop = QStandardPaths::writableLocation(QStandardPaths::DesktopLocation);
+    auto executable_path = APPLICATION->applicationFilePath();
+    auto instId = m_selectedInstance->id();
+    auto icon = APPLICATION->windowIcon();
+    auto name = m_selectedInstance->name();
+#if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
+    createLink(
+        const_cast<wchar_t *>(executable_path.toStdWString().c_str()),
```

Comment:
Any reason for the const_casts here? Why not make the LPCWSTR arguments const refs?